mlflow/server/graphql/autogenerated_graphql_schema.py

```diff
@@ -0,0 +1,103 @@
+# GENERATED FILE. PLEASE DON'T MODIFY.
+# Run python3 ./dev/proto_to_graphql/code_generator.py to regenerate.
+import graphene
+
+import mlflow
+from mlflow.server.graphql.graphql_custom_scalars import LongString
+from mlflow.utils.proto_json_utils import parse_dict
+
+
+class MlflowRunStatus(graphene.Enum):
+    RUNNING = "RUNNING"
+    SCHEDULED = "SCHEDULED"
+    FINISHED = "FINISHED"
+    FAILED = "FAILED"
+    KILLED = "KILLED"
+
+
+class MlflowDataset(graphene.ObjectType):
+    name = graphene.String()
+    digest = graphene.String()
+    source_type = graphene.String()
+    source = graphene.String()
+    schema = graphene.String()
+    profile = graphene.String()
+
+
+class MlflowInputTag(graphene.ObjectType):
+    key = graphene.String()
+    value = graphene.String()
+
+
+class MlflowDatasetInput(graphene.ObjectType):
+    tags = graphene.List(graphene.NonNull(MlflowInputTag))
+    dataset = graphene.Field(MlflowDataset)
+
+
+class MlflowRunInputs(graphene.ObjectType):
+    dataset_inputs = graphene.List(graphene.NonNull(MlflowDatasetInput))
+
+
+class MlflowRunTag(graphene.ObjectType):
+    key = graphene.String()
+    value = graphene.String()
+
+
+class MlflowParam(graphene.ObjectType):
+    key = graphene.String()
+    value = graphene.String()
+
+
+class MlflowMetric(graphene.ObjectType):
+    key = graphene.String()
+    value = graphene.Float()
+    timestamp = LongString()
+    step = LongString()
```

Comment:
for compatibility with current REST API, can we retrieve those values as integers?

Reply:
also it's int64 in protofiles: https://src.dev.databricks.com/mlflow/mlflow/-/blob/mlflow/protos/service.proto?L478-482

Reply:
Synced offline, this is for keeping consistency with the in house graphql schema we have